build/debs/BUILD

```diff
@@ -164,6 +164,7 @@ k8s_deb(
     description = """Kubernetes Packaging of CNI
 The Container Networking Interface tools for provisioning container networks.
 """,
+    preinst = "kubernetes-cni.preinst.sh",
```

Comment:
I'm not a debian packaging expert.  /cc @marcoceppi @castrojo 

With rpm.specs you own the directories in the spec.